classes/import-export.php

```diff
@@ -826,7 +822,7 @@ public function print_page() {
 				_e( 'Import CSV File', 'SimpleMap' );
 				echo '"/>';
 				if ( '' == $options['api_key'] ) : ?>
-															   <?php printf( __( "Warning: You still need to enter an <a href='%s'>API key</a> if you need your locaitons geocoded.", 'SimpleMap' ), admin_url( "admin.php?page=simplemap" ) ); ?>
+															   <?php printf( __( "Warning: You still need to enter an <a href='%s'>API key</a> if you need your locations geocoded.", 'SimpleMap' ), admin_url( "admin.php?page=simplemap" ) ); ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected 64 spaces, found 63
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 242 characters

<!-- Comment by [SideCI](https://sideci.com) -->